server_src/routes/printers.js

```diff
@@ -125,25 +127,26 @@ router.get("/groups", ensureAuthenticated, async (req, res) => {
 });
 router.post("/printerInfo", ensureAuthenticated, async (req, res) => {
   const id = req.body.i;
+  const returnPrinter = await PrinterClean.returnPrintersInformation(id);
+  res.send(returnPrinter);
+});
```

Comment:
Simplified the code with the new allowance of an ID check within return printers information